nav2_rviz_plugins/include/nav2_rviz_plugins/nav2_panel.hpp

```diff
@@ -182,10 +190,12 @@ private Q_SLOTS:
   QLabel * imgDisplayLabel_{nullptr};
 
   // The following states are added to allow for the state of the button to only expose reset
-  // while the NavigateToPoses action is not active. While running, the user will be allowed to
-  // cancel the action. The ROSActionTransition allows for the state of the action to be detected
+  // while neither the NavigateToPoses action nor FollowWaypoints are active. While running,
+  // the user will be allowed to cancel the action.
+  // The ROSActionTransition allows for the state of the action to be detected
   // and the button state to change automatically.
-  QState * running_{nullptr};
```

Comment:
on first place, why is this being split into two different states? `running_` is kind of the state that the state machine transition to, when you give a single goal. This is no ways related to the waypoint navigation.

The states `accumulated_wp_` and `accumulated_nav_through_poses_` are the equivalent to the `running_` state for the navigation in the waypoint mode.